spec/acceptance/support/puppetdb.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+def has_puppetdb
+  case host_inventory['facter']['os']['name']
+  when 'Debian'
+    return false if ENV['BEAKER_PUPPET_COLLECTION'] == 'puppet6' && host_inventory['facter']['os']['release']['major'] == '11'
+  end
+
+  true
+end
```

Comment:
This should be `spec/support/acceptance/puppetdb.rb` :wink:

Reply:
I copied the pattern from the foreman example @ekohl liked to, I like the idea of isolating actual test code from support code that could be in a shared gem if it was used by more projects.

Would this cause trouble?